docs/attributes-registry/system.md

```diff
@@ -0,0 +1,160 @@
+# System
+
+<!-- toc -->
+
+- [CPU attributes](#cpu-attributes)
+- [Memory attributes](#memory-attributes)
+- [Paging attributes](#paging-attributes)
+- [Filesystem attributes](#filesystem-attributes)
+- [Network attributes](#network-attributes)
+- [Process attributes](#process-attributes)
+
+<!-- tocstop -->
```

Comment:
The GitHub viewer gives you a TOC automatically for markdown files (as other viewers should offer as well).

Hard-coding a TOC has the danger of easily getting out of sync with follow-up changes to the doc. It's another thing you have to maintain and think of when contributing.

IMO, it's better to not have a TOC. Or maybe there is an agreement/policy on that!?

Reply:
This TOC is being generated through the build tools and checks are being executed as part of CI to make sure it doesn't get outdated.

Reply:
@rockdaboot the toc generation is created automatically if tags are provided and is checked for every PR as a part of workflow (`markdown-toc`)

Reply:
Thanks :+1: (just found your answers in my private mbox which I check only on weekends)